joplin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -0,0 +1,49 @@
+{% load i18n %}
+{% if page_types_restricted %}
+    {% if page_type_string == "base.departmentpage" %}
+        {% trans "Add a department that offers this page" as choose_str %}
+    {% elif page_type_string == "base.topiccollectionpage" %}
+        {% trans "Select a topic collection" as choose_str %}
+    {% elif page_type_string == "base.topicpage" %}
+        {% trans "Select a topic " as choose_str %}
+    {% elif page_type_string == "base.informationpage,base.servicepage" %}
+        {% trans "Select a page for the guide" as choose_str %}
+    {% elif page_type_string == "base.informationpage,base.servicepage,base.guidepage,base.officialdocumentpage" %}
+        {% trans "Choose content to feature on the department page" as choose_str %}
```

Comment:
I understand why this exists this way now, but this logic will be messy when we need to have the same page_type_string for our picker on the editable home page.

We should find a better way to determine we're definitely "choosing content to feature on the department page"

Reply:
I agree this is hacky and merely deductive. I could not find a better way other than using generic page chooser. The page type of the page you are on isn't passed to the template and there's little else to go off of for conditionals. 

Meanwhile the generic chooser makes it pretty easy to define custom page choosers with their own templates and titles, so we won't even need this nest of conditionals then.

Reply:
sounds like the issue to make is: "refactor custom page choosers to have their own templates and titles"

Reply:
https://github.com/cityofaustin/techstack/issues/3016